### DISCUSSION TITLE
Compute cc - error with SAC files

### DISCUSSION THREAD

**Discussion by Amituofuo** [ROBelgium/MSNoise#276] [Q&A] [answered]
Hi 
I am a beginner of seismic processing and MSnoise. I have many questions to discuss with you. When I run msnoise - v compute cc, DEBUG prompts as follows：
msnoise [pid 10184][INFO]: *** Starting: Compute CC ***
msnoise [pid 10184][INFO]: Will compute [ZZ] for different stations
msnoise [pid 10184][INFO]: Will compute [] for single stations
msnoise [pid 10184][INFO]: Checking if there are jobs to do
msnoise [pid 10184][INFO]: Getting the next job
msnoise [pid 10184][INFO]: New CC Job: 2022-07-16 (1 pairs with 2 stations)
msnoise [pid 10184][DEBUG]: F.1339.Z Reading 1 Files
msnoise [pid 10184][DEBUG]: ERROR reading file G:/Win-Linux/16-19\1339.16-19.z.1624.SAC
msnoise [pid 10184][DEBUG]: F.1340.Z Reading 1 Files
msnoise [pid 10184][DEBUG]: ERROR reading file G:/Win-Linux/16-19\1340.16-18.Z.1624.SAC
msnoise [pid 10184][DEBUG]: Not enough data for this day !
msnoise [pid 10184][DEBUG]: Marking job Done and continuing with next !
msnoise [pid 10184][INFO]: *** Finished: Compute CC ***
ThomasLecocq explained that this error occurs for a single "read" command from obspy, so you can try reading the file again with obspy and check the traceback. But how can I try reading the file again with obspy and check the traceback.

**Comment by ThomasLecocq** [comment, marked as answer]
I suggest you take some time to go through the ObsPy tutorials: https://docs.obspy.org/tutorial/index.html

About your issue, I suspect it is due to the fact that you don't have all headers defined in the SAC header (NET, STA, LOC, CHAN) and/or that you didn't define the ``network`` configuration value in the config, etc...

**Reply by Amituofuo** [reply]
Thank you Thomas Lecocq, I have successfully calculated dv\v. As you said, the lack of SAC header makes it impossible to calculate. I have a lot to learn about seismic processing. I also have a question for you to ask how the station coordinates are imported. Is it edited by header or entered directly at'MSnoise'- station?

**Reply by ThomasLecocq** [reply]
it is entered either manually in the station table, or if you have a response/inventory file(s), you can ``msnoise config --sync`` to input the coordinates to the table.
